mkdocs.yml

```diff
@@ -0,0 +1,47 @@
+site_name: AWS .NET deployment tool
+site_description: 'Deploy .NET applications on AWS'
+site_url: 'https://aws.github.io/aws-dotnet-deploy/'
+repo_name: 'aws/aws-dotnet-deploy'
+repo_url: 'https://github.com/aws/aws-dotnet-deploy'
+edit_uri: 'edit/main/site/content'
+copyright: '&copy; 2022, Amazon Web Services, Inc. or its affiliates. All rights reserved.'
```

Comment:
Is the "2022" required here? Per the internal wiki for source code we've been moving to just (though unsure if this is different since it's a website)
> “Copyright Amazon.com Inc. or its affiliates.”

Reply:
This more of a question for management/legal. I have seen the year attribute being included other AWS open sourced projects.

https://awslabs.github.io/aws-lambda-powertools-python/latest/
https://aws.github.io/copilot-cli/

We can keep the "2022" for now and change it later if required.

Reply:
Sounds good, it's on our current docs site too - https://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/welcome.html